task_1/style.css

```
@@ -8,6 +8,147 @@
     src: url(./Fonts/Product_Sans/Product\ Sans\ Regular.ttf);
 }
 
+.wrapper {
+    background: linear-gradient(180deg, rgba(228, 239, 233, 0) 0%, #ECF8F1 6.25%);
+    margin: 0 auto;
+    max-width: 1440px;
+}
+
+.nav-list {
+    list-style: none;
+    margin: 20px 40px;
+    text-align: center;
+    width: 90%;
+}
+
+#store-image {
+    float: left;
+    justify-self: end;
+}
+
+#profile-image {
+    float: right;
+    margin: 0;
+    padding: 0px;
+}
+
+.store-size {
+    margin: 7px;
+    max-height: 40px;
+    max-width: 45px;
+    position: relative;
+}
+
+.profile-size {
+    margin: 5px 0 0 0;
+    max-height: 60px;
+    max-width: 60px;
+}
+
+.cart-size {
+    float: right;
+    margin: 10px;
+    max-height: 40px;
+    max-width: 40px;
+    padding: 0px;
+}
+
+.hover-cursor{
+    cursor: pointer;
+}
+
+li {
+    display: inline;
+    font-family: OpenFont;
+    margin: 10px;
+}
+
+li:hover {
+    color: #EB5757;
+}
+
+.active {
+    color: rgb(97, 97, 146);
+}
+
+.red-color {
+    color: #EB5757;
+    font-size: 20px;
+    font-weight: bold;
+    margin: 0;
+}
+
+.black-color {
+    font-size: 40px;
+    margin: 0;
+}
```

the color property missing depend on the class name 
```suggestion
.black-color {
    color: black;
    font-size: 40px;
    margin: 0;
}
```